.github/workflows/release.yaml

```diff
@@ -0,0 +1,29 @@
+# .github/workflows/release.yaml
+on:
+  release:
+    types: [created]
+
+jobs:
+  releases-matrix:
+    name: Release Binaries
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # build and publish in parallel: linux/386, linux/amd64, linux/arm64, windows/386, windows/amd64, darwin/amd64, darwin/arm64
+        goos: [linux, windows, darwin]
+        goarch: ["386", amd64, arm64]
+        exclude:
+          - goarch: "386"
+            goos: darwin
+          - goarch: arm64
+            goos: windows
+    steps:
+    - uses: actions/checkout@v2
+    - uses: wangyoucao577/go-release-action@v1.26
```

Comment:
Please pin to hash for non-official actions (`actions/`, docker ones for docker, etc.)

Of course, ideally we'd _not_ use third party actions, and just do `setup-go` + build + `gh` or alternative.